src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Public.Data/DataSetVersionsControllerTests.cs

```diff
@@ -42,7 +42,7 @@ public abstract class DataSetVersionsControllerTests(
 {
     private const string BaseUrl = "api/public-data/data-set-versions";
 
-    public class ListLiveVersionsTests(
+    public class ListVersionsTests(
```

Comment:
Should we rename this back to `ListLiveVersionsTests` as we've kept the logic to only return the live ones?

Reply:
I made this change on purpose as it felt too specific a name for an endpoint that lists versions.

IMO, filtering on 'live' statuses makes sense as a default, but it'd be a good idea to keep the naming more generic to keep the endpoint open to extension for other statuses (e.g. with a query parameter or something) in the future.

Reply:
I don't feel too strongly either way. Happy to approve :)